src/index.ts

```diff
@@ -0,0 +1,4 @@
+import { serve } from '@hono/node-server'
+import { userRoute } from './routes'
+
```

Comment:
Here in the index.ts you need to create the app and import routes here like this
```
const app = new Hono();
app.route('/', routes);
```